engine/src/test/java/org/camunda/bpm/engine/test/api/authorization/ManagementAuthorizationTest.java

```diff
@@ -252,14 +252,17 @@ public void testTelemetryEnabledWithoutAutorization() {
 
   public void testTelemetryEnabledAsCamundaAdmin() {
     // given
+    disableAuthorization();
+    managementService.enableTelemetry(true);
+    enableAuthorization();
     identityService.setAuthentication(userId, Collections.singletonList(Groups.CAMUNDA_ADMIN));
 
     // when
-    managementService.enableTelemetry(true);
+    managementService.enableTelemetry(false);
```

Comment:
Not directly related to the topic of this ticket: I think this API design is a bit confusing. To disable delemetry, I would write `managementService.enableTelemetry(false)` which is not intuitive to read. Maybe `toggleTelemetry` instead of `enableTelemetry` is more clear.

Reply:
I will change it CAM-12078